PostBoard.Api/Controllers/BirthdayController.cs

```diff
@@ -51,15 +50,16 @@ public async Task<IActionResult> CreateAsync([FromBody] Birthday input)
         {
             return BadRequest("Validation error.");
         }
-        await _dbcontext.Birthdays.AddAsync(input);
-       await _dbcontext.SaveChangesAsync();
-
-        return Ok(input);
+        var birthdayId = await _birthdayService.CreateAsync(input);
+        
+        
+        // ReSharper disable once Mvc.ActionNotResolved
+        return CreatedAtAction(nameof(GetByIdAsync), new { id = birthdayId }, input);
     }
 
     [HttpPut]
     [Route("{id:int}")]
-    public  async Task<IActionResult>  UpdateAsync([FromRoute] int id, [FromBody] Birthday input)
+    public  async Task<IActionResult>UpdateAsync([FromRoute] int id, [FromBody] Birthday input)
```

Comment:
Add space ` ` before `UpdateAsync`. You had 2 spaces earlier, but only 1 is needed)